app/components/screens/ClientQuests.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, ScrollView, ActivityIndicator, Text, View, Dimensions, Image, Pressable, FlatList, TouchableOpacity} from 'react-native';
+import {useNavigation} from '@react-navigation/native'
+import {FontAwesome, Entypo} from '@expo/vector-icons'
+import Config from '../../../config.json'
+import Tags from "react-native-tags"
+
+const {width} = Dimensions.get('screen')
+
+export default ClientQuests = ({route, navigation}) => {
+  //  const navigation = useNavigation()
+
+    const {ID, name, image} = route.params
+
+    const [data, setData] = useState([])
+    const [tags, setTags] = useState(['hola'])
+    const [loading, setLoading] = useState(true)
+
+    const url = Config.appUrl + "clients/" + ID + "/quests"
+
+    const getData = () => {
+        if (loading) {
+            return <ActivityIndicator size="large" />
+         }
+         return (
+            data.map( (quest, index) => 
+            <Pressable key={index}>
+                <View  style={styles.optionCard} key = {index}>
+                <Text style={{textAlign: 'center', marginTop:10, fontSize:18, fontWeight: 'bold'}}>{quest.name}</Text>
+                </View>        
+            </Pressable>
+            )
+         )
+    
+    }
+
+    useEffect(() => {
+        fetch(url)
+        .then((response) => response.json())
+        .then((json) => setData(json))
+        .catch((error) => console.error(error))
+        .finally(()=>setLoading(false))
+    }, [])
+
+    const Card = ({quest}) => {
```

Comment:
Esto es una de ejemplo hardcodeada no?

Reply:
ahi saqué el hardcodeo de los tags, me habia quedado pendiente de ayer

Reply:
Vas a tener que perdonar que esta pantalla esta pensada para que las busquedas tengas 4 tags jajajajajaja, nice to have: cantidad de tags variables

Reply:
No se podria hacer directamente para muchos tags pero nosotros tenemos cuidado de no meterle mas de 4? Cuál seria el problema de los tiempos? Es muy lento?